.changeset/bright-bags-align.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/evals': minor
+---
+
+Add opt-in conversation context to the Prompt Alignment scorer for multi-turn Agent evaluations.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)